pkg/model/deployment.go

```diff
@@ -48,6 +48,11 @@ func IsCompletedDeployment(status DeploymentStatus) bool {
 	return false
 }
 
+// IsCompletedSuccessfullyDeployment checks whether the deployment is successfully addressed.
+func IsCompletedSuccessfullyDeployment(status DeploymentStatus) bool {
```

Comment:
nit: I feel like this is more natural but it's on you
```suggestion
func IsSuccessfullyCompletedDeployment(status DeploymentStatus) bool {
```

Reply:
okay, lets me adopt your suggestion 🙆

Reply:
addressed by https://github.com/pipe-cd/pipe/pull/2832/commits/640e1726debbcd21c06055e93f8b06d5c5303ed6